lstchain/reco/__init__.py

```diff
@@ -1,4 +1,6 @@
-from .utils import get_event_pos_in_camera
-from .r0_to_dl1 import get_dl1
-from . import dl1_to_dl2
+from .utils import *
+from .r0_to_dl1 import get_dl1, add_disp_to_parameters_table
+from .dl1_to_dl2 import *
+from .volume_reducer import *
+from .disp import *
```

Comment:
Please no star imports without providing `__all__`. Neither editors nor code linters will be able to work properly (autocompletion / linting)

Reply:
I'm not sure what do you mean, all files provide `__all__`, but some functions were missing though, this is what you mean?

Reply:
This file does not have an `__all__`.

Reply:
ok, I get it. We need to modify all the `__init__` files because none have `__all__` and are making wildcard imports. Also modify some of the calls to avoid
> Also, I had some troubles with the __init__ because the file r0_to_dl1.py contains the function r0_to_dl1 (amongst others), then it cannot be included in the init as:
from .r0_to_dl1 import *

can we merge this one and I open an Issue/PR right afterwards to solve that in a different thread?

Reply:
so @MaxNoe shall we move on with this PR and I'll correct the different `__init__` files afterwards?

Reply:
ok